packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/__snapshots__/react-ssr-wrapper.spec.tsx.snap

```diff
@@ -949,7 +949,7 @@ img {
               </style>
               <img
                 alt=""
-                src="https://cdn.ui.porsche.com/porsche-design-system/icons/plus.min.dd34f71292a547080b9247a72c1adda2.svg"
+                src="http://localhost:3001/icons/plus.min.d5572392a68e615c6b5fb0ccbba5ef59.svg"
```

Comment:
I guess it would make sense to extend our smoke tests to verify that the production build of the ssr sub-package does not contain localhost.